buildchain/buildchain/image.py

```diff
@@ -249,6 +249,24 @@ def show() -> str:
         destination=constants.ISO_IMAGE_ROOT,
         task_dep=['_image_mkdir_root'],
     ),
+    targets.RemoteImage(
+        registry=constants.METALLB_REGISTRY,
+        remote_name='controller',
+        name='metallb-controller',
+        version='v0.7.3',
```

Comment:
Minor: we could have a constant `LLB_VERSION : str = 'v0.7.3'` so that when we bump the version we have a single replace instead of several.